### DISCUSSION TITLE
User interface design

### DISCUSSION THREAD

**Discussion by archienorman11** [aptakube/aptakube#6] [General]
I’m curious to know if you used an OS design system for the user interface. It looks great! 

Many thanks

**Comment by goenning** [comment]
hey @archienorman11 , thanks for the feedback, happy to know you liked!

It's not an off the shelf design system, the UI is built with Tailwind and DaisyUI, but a lot of the inspiration came from TailwindUI, Vercel, Linear, Logtail and lately I've been looking at a lot of the macOS guidelines as well

**Comment by archienorman11** [comment]
Interesting! Im a huge Linear / Logtail / TailwindUI fan. We use TailwindUI for a lot of projects but I haven't made the jump to daisy UI yet. We decided to test out creating our own component library from the TailwindUI components (basically just creating smaller components). Has DaisyUI worked well for you?

**Reply by goenning** [reply]
I used it a lot more when I started as it was quicker, but as the project grew, I opted for hand made components. At the moment I only use it for themes and I might eventually drop it in future
